lms/djangoapps/mobile_api/preferences/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
@DanielEidan  You can see the Test results on this PR by clicking on the "Details" link below and then following through on the "Build Graph" link.  You can get more information in the following document: https://openedx.atlassian.net/wiki/display/TE/Jenkins+Guide
Feel free to reach out if you have further questions on this.